react/features/base/config/configType.ts

```diff
@@ -440,6 +440,7 @@ export interface IConfig {
     };
     localSubject?: string;
     locationURL?: URL;
+    mainToolbarButtons?: Array<Array<ToolbarButton | string>>;
```

Comment:
Generally yes, since ToolbarButton is ultimately a string. But when we add the ToolbarButton type this can improve the autocomplete in the IDE, you get some suggestions from ToolbarButton type. In addition it looks a little but clearer to me because it shows that you can put our stock buttons or a custom one (which is represented by `| string`). 

If you don't like it I don't mind removing the `ToolbarButton` part tho.

Reply:
Done!